### PR TITLE
Signup throttling

### PR DESCRIPTION
**Pull request by dylanlott**

### CODE REVIEW

**Comment by dylanlott** [comment]
Implement delayed activation on user sign up.

**Comment by braydonf** [comment]
We should have a config option to disable/enable this for different environments, e.g. staging/testing.

File: package.json [hunk 1]
```diff
@@ -85,7 +85,7 @@
     "storj-lib": "^8.0.0",
     "storj-mongodb-adapter": "^9.0.0",
     "storj-service-error-types": "^1.4.0",
-    "storj-service-mailer": "^1.0.0",
+    "storj-service-mailer": "Storj/service-mailer",
```
Comment (braydonf) [inline package.json]: Mailer needs to be published?

**Comment by bryanchriswhite** [comment]
@braydonf yes service-mailer will need to be published after we test in staging.

**Pull request merged**